### ISSUE TITLE
Font sizes redifinitions

### ISSUE THREAD

**Issue by andreev-artem** [salesforce-ux/design-system#142]
In 0.12.1 there are font size re-definitions in ```design-tokens.scss```:

```
$font-size-x-small: 0.625rem !default;
$font-size-small: 0.875rem !default;
$font-size-medium: 1rem !default;
$font-size-large: 1.25rem !default;
$font-size-x-large: 1.5rem !default;
$font-size-xx-large: 2rem !default;
```
at the lines 143-148
and below at the lines 527-529:
```
$font-size-small: 0.75rem;
$font-size-medium: 0.875rem;
$font-size-large: 1.125rem;
```

That introduces some inconsistency and now, for example, I can't set 1rem size to a title using design token.

Is that bug?

**Comment by kaelig** [comment]
Thank you for using the Salesforce Lightning Design System, @andreev-artem.

It does look like a bug in the zip version of the SLDS, thank you for bringing this to our attention.

We'll get back to you shortly, thank you for your patience.

**Comment by andreev-artem** [comment]
That exists also in bower:
https://github.com/salesforce-ux/design-system-bower/blob/master/scss/design-tokens.scss#L143-L148
https://github.com/salesforce-ux/design-system-bower/blob/master/scss/design-tokens.scss#L527-L529

**Comment by kaelig** [comment]
@andreev-artem thank you again for flagging this up, we really appreciate it. This will be fixed in the next release, thank you for your patience.

**Comment by andreev-artem** [comment]
It doesn't seem to be fixed in 0.12.2. Though some things were changed - now wrong definition is placed earlier than definition with value from docs. And because the latter one uses !default it doesn't redefine the wrong value.

https://github.com/salesforce-ux/design-system-bower/blob/master/scss/design-tokens.scss#L12
https://github.com/salesforce-ux/design-system-bower/blob/master/scss/design-tokens.scss#L172

According the docs $font-size-medium should be 1rem.

The same situation exists for $line-height-text (according the docs it should be 1.375):
https://github.com/salesforce-ux/design-system-bower/blob/master/scss/design-tokens.scss#L14
https://github.com/salesforce-ux/design-system-bower/blob/master/scss/design-tokens.scss#L177

**Comment by andreev-artem** [comment]
@kaelig, do you think the bug still exists?

**Comment by kaelig** [comment]
@andreev-artem this is fixed in the next imminent release, not in 0.12.x. Thank you again for your patience.
